tests/coverage/foundationals.md

```diff
@@ -0,0 +1,819 @@
+# Test coverage of SpeechBrain
```

Comment:
The coverage folder contains too much Markdown files. We should break this down to something more comprehensive. In the current form, users will never end up on those resources. I think this should be put into 4 markdown AT MOST.

Reply:
That one was my original one long md file from where I derived the others. I let it in as it contains outlines & markers for a proof that all is covered by the current testing & what is potentially missing. An outcome of that exercise was also the future ideas file. It's only of interest of scholarly archivers now, I think—it can go.

@TParcollet with the other changes you suggested, the 4 md files at most would be easily achieved.

Reply:
done (removed this one)